static/js/app.js

```diff
@@ -84,13 +84,24 @@ const shortcutsTarget = $('#shortcuts');
 if (shortcutsTarget.length > 0) {
   $('.content-container h2, .content-container h3').map(function(idx, el) {
     const title = el.textContent;
+    // transforms title into snake-case
+    const elTitle = title.replace(/\s/g, '-').toLowerCase();
+    // Gets the element type (e.g. h2, h3)
     const elType = $(el).get(0).tagName;
-    shortcutsTarget.append(`<div id="${title}-shortcut" class="shortcuts-${elType}">${title}</div>`);
+    // Adds snake-case title as an id attribute to target element
+    $(el).attr('id', elTitle);
+    shortcutsTarget.append(`<div id="${elTitle}-shortcut" class="shortcuts-${elType}">${title}</div>`);
 
-    $(`#${title}-shortcut`).click(function() {
+    $(`#${elTitle}-shortcut`).click(function() {
       $([document.documentElement, document.body]).animate({
-        scrollTop: $(`#${title}`).offset().top
+        scrollTop: $(`#${elTitle}`).offset().top-60
     }, 1000);
     })
   });
+}
+
+// Removes the shortcuts container if no shortcuts exist.
+// Also removes the 'Get Help' link.
+if ($('#shortcuts div').length < 1) {
+  $('.shortcuts-container').css('display', 'none');
```

Comment:
I'm not sure this is wanted. Curious everyone's thoughts here. Another option might be to simply remove the 'Shortcuts' title, however, i think the `Get Help?` link might want to live somewhere else so it's not the only element on the left sidebar.

Reply:
I agree, it's kind of weird to have the sidebar with a single link. This looks better. Let's merge and deal with the get help separately - it could live in the footer for example